package.json

```diff
@@ -16,7 +16,7 @@
     "jslint": "eslint lib tests scripts bin versionist.conf.js",
     "sasslint": "sass-lint lib/gui/scss",
     "htmllint": "node scripts/html-lint.js",
-    "codespell": "codespell.py --skip *.gz,*.bz2,*.xz,*.zip,*.img lib tests docs scripts Makefile *.md LICENSE",
+    "codespell": "codespell --dictionary=- --dictionary=dictionary --skip *.gz,*.bz2,*.xz,*.zip,*.img lib tests docs scripts Makefile *.md LICENSE",
```

Comment:
How is this change related to this PR? Appveyor CI is throwing:

> 'codespell' is not recognized as an internal or external command,
operable program or batch file.